### PR TITLE
Included Scala.IO 2016 in the list of Scala events

### PR DESCRIPTION
**Pull request by jeantil**

### CODE REVIEW

**Comment by heathermiller** [comment]
Cool! Happy to hear that it's happening this year!! :) LGTM, thanks for posting it

**Comment by jeantil** [comment]
Wow that's a quick merge, thanks a lot ! 

We are happy about it too and notice how we brought it closer to lausanne ;p

**Comment by heathermiller** [comment]
True :) hmm

File: events/_posts/2016-10-27_scalaio.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+category: event
+title: Scala.IO 2016
+logo: /resources/img/scala-io.png
+location: Lyon, France
+description: Begginer to advanced user  Scala and FP conference in france
```
Comment (SethTisue) [inline events/_posts/2016-10-27_scalaio.md]: s/Begginer/Beginner/

Comment (SethTisue) [inline events/_posts/2016-10-27_scalaio.md]: s/france/France/

Comment (jeantil) [inline events/_posts/2016-10-27_scalaio.md]: Sorry about that, fixed in #412

**Pull request merged**